pkg/helm/helm_helpers.go

```diff
@@ -596,11 +596,23 @@ type HelmRepoCredential struct {
 
 // DecorateWithSecrets will replace any vault: URIs with the secret from vault. Safe to call with a nil client (
 // no replacement will take place).
-func DecorateWithSecrets(options *InstallChartOptions, secretURLClient secreturl.Client) (func(), error) {
+func (options *InstallChartOptions) DecorateWithSecrets(secretURLClient secreturl.Client) (func(), error) {
+	newValuesFiles, cleanup, err := DecorateWithSecrets(options.ValueFiles, secretURLClient)
+	if err != nil {
+		return cleanup, errors.WithStack(err)
+	}
+	options.ValueFiles = newValuesFiles
+	return cleanup, nil
+}
+
+// DecorateWithSecrets will replace any vault: URIs with the secret from vault. Safe to call with a nil client (
+// no replacement will take place).
+func DecorateWithSecrets(valueFiles []string, secretURLClient secreturl.Client) ([]string, func(), error) {
```

Comment:
nit: make this private if is not used by other packages

Reply:
It is used by another package, see https://github.com/jenkins-x/jx/pull/5480/files#diff-939ef26a208151dd89ac7bdd1ac1741dR337